src/connect/order/Order.ts

```diff
@@ -5,7 +5,6 @@ import { NewAddress } from './Address';
 import { Company } from './Customer';
 
 export type Order = {
-    id: string;
```

Comment:
I think leaving this out of the type helps users. But feel free to change my mind.